packages/components/src/header/index.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Button, Icon, DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { help, wordpress, moreVertical } from '@wordpress/icons';
+import Header from './index';
+
+const meta = {
+	title: 'Header',
+	component: Header,
```

Comment:
@mirka I think the direction is to have a flat storybook and that's the reason for this line. Please correct me if I'm wrong.

Reply:
The component itself needs a more descriptive (or namespaced) name though, no? `Header` is extremely broad.

Reply:
You're right about it being very broad, but I couldn't think of something better myself (the name come from design) without prefixing it indeed. Any suggestions here are welcome. --cc @jameskoster @jasmussen @matt-west

Reply:
maybe `PageHeader`

Reply:
>maybe PageHeader

This component is going to be used in sections too.

Reply:
> This component is going to be used in sections too

This is how we set up the component in Figma, but it doesn't need to be this way in the code. `PageHeader` and `SectionHeader` abstractions might be helpful because Page headers have some unique aspects like stickiness.

Reply:
How does affect the naming though? Are you saying that we could rename this to `PageHeader` and have a small wrapper for this component for sections (or the other way around - rename to SectionHeader etc..)?

>Page headers have some unique aspects like stickiness.

I'm not aware of this. Are you thinking about a prop in the component?

Reply:
The naming question seems like an engineering decision. Design will follow whatever configuration y'all feel works best.

> Are you thinking about a prop in the component?

If that seems like a good way to codify it, sure.

Reply:
I'm going to rename to `PageHeader` for now. Alternatives I thought were: `SectionHeader, ContentHeader, TitleBar`. I think `PageHeader` communicates better the purpose of the component while not being too broad.